### PR TITLE
feat(api): use GET requests and CDN host for content service

### PR DESCRIPTION
**Pull request by mhaack**
Change API worker to use:
* GET requests for GraphQL queries on the content service
* use the inner CDN
* remove model KV usage in favor of inner CDN

[SITES-6761](https://jira.corp.adobe.com/browse/SITES-6761)

### CODE REVIEW

**Comment by icaraps** [comment]
You can also delete the MODEL KV reference in the [wrangler.toml](https://github.com/adobe-rnd/franklin/blob/main/packages/franklin-api/wrangler.toml#L6) file (and from the CF dashboard once PR is merged)

**Review by icaraps — APPROVED** [review]

File: packages/franklin-api/src/index.js [hunk 1]
```diff
@@ -64,60 +64,32 @@ router.get('/model', async (request, env) => {
...
-  }
-
-  if (!project) {
-    return corsResponse(`Missing parameter "project"`, { status: 400 });
+  if (reqModel.ok) {
+    const resModel = await reqModel.json();
+    model = resModel?.data?.documents[0] ?? {};
   }
 
-  model = await env.MODELS.get(`${project}--${tenant}--${path}.hast.json--${preview}`, { type: 'stream' });
```
Comment (icaraps) [inline packages/franklin-api/src/index.js:114]: @mhaack  I just noticed we don't support previewing models anymore with this change ... ie how can you have multiple variations of a model at a given path ?

Comment (icaraps) [reply packages/franklin-api/src/index.js:114]: So I did this https://github.com/adobe-rnd/franklin/commit/f6382730af5e14e3d6f034ba4f69ab2bdd8175fc

Comment (mhaack) [reply packages/franklin-api/src/index.js:114]: Thanks

**Pull request merged**